app/models/gsa18f/proposal_form.rb

```diff
@@ -0,0 +1,87 @@
+module Gsa18f
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+
+  class ProposalForm
+    include SimpleFormObject
+
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    
+    attribute :requester, :user
+    attribute :office, :string
+    attribute :justification, :text
+    attribute :link_to_product, :string
+    attribute :quantity, :integer
+    attribute :date_requested, :datetime
+    attribute :urgency, :string
+    attribute :additional_info, :string
+    attribute :cost_per_unit, :decimal
+    attribute :product_name_and_description, :text
+    attribute :origin, :string
+
+    validates :cost_per_unit, numericality: {
+      greater_than_or_equal_to: 0,
+      less_than_or_equal_to: 3000
+    }
+    validates :quantity, numericality: {
+      greater_than_or_equal_to: 1
+    }
+    validates :product_name_and_description, presence: true
+
+    def set_approver_on(cart)
+      cart.add_approver(ENV['GSA18F_APPROVER_EMAIL'] || 'Richard.L.Miller@gsa.gov')
```

Comment:
Let's not have real email addies in the code base; instead, make this a dummy like `18fapprover@gsa.gov`